chain/jsonrpc/src/api/transactions.rs

```diff
@@ -61,3 +61,51 @@ fn decode_signed_transaction(value: Value) -> Result<SignedTransaction, RpcParse
     SignedTransaction::try_from_slice(&bytes)
         .map_err(|err| RpcParseError(format!("Failed to decode transaction: {}", err)))
 }
+
+#[cfg(test)]
+mod tests {
+    use crate::api::RpcRequest;
+    use near_jsonrpc_primitives::types::transactions::{
+        RpcBroadcastTransactionRequest, RpcTransactionStatusCommonRequest,
+    };
+    use near_primitives::borsh::BorshSerialize;
+    use near_primitives::hash::CryptoHash;
+    use near_primitives::serialize::to_base64;
+    use near_primitives::transaction::SignedTransaction;
+    use std::str::FromStr;
+
+    #[test]
+    fn test_serialize_tx_status_params_as_vec() {
+        let tx_hash = "6zgh2u9DqHHiXzdy9ouTP7oGky2T4nugqzqt9wJZwNFm";
```

Comment:
nit: I would either generate a random one on the fly or extract it to a constant. This removes those long hashes from the code and makes it easier to read.